### PR TITLE
SAML Grafana guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Documents using Teleport SAML to authenticate to Grafana Enterprise.

### CODE REVIEW

**Review by mdwn — APPROVED** [review]

File: docs/pages/access-controls/idps/saml-grafana.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+SAML authentication. This guide will help you configure Teleport as a SAML provider,
+and Grafana to accept the identities it provides.
+
+Note that Teleport can act as an identity provider to any SAML-compatible service,
+not just those running behind the Teleport App Service.
+
+## Prerequisites
+
+- An instance of Grafana Enterprise, with edit access to `grafana.ini`.
+  - A certificate solution to create TLS certificates/keys for the SAML connection.
```
Comment (zmb3) [inline docs/pages/access-controls/idps/saml-grafana.mdx:16]: Can we be more precise here? A quick google of _certificate solution_ pulls up a ton of unrelated items.

File: docs/pages/access-controls/idps/saml-grafana.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    rules:
+    - resources:
+      - saml_idp_service_provider
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+version: v6
```
Comment (zmb3) [inline docs/pages/access-controls/idps/saml-grafana.mdx:44]: I find it easier to read if `version` is up with kind and metadata as opposed to after the spec.

**Review by zmb3 — APPROVED** [review]

File: docs/pages/access-controls/idps/saml-grafana.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/access-controls/idps/saml-grafana.mdx]: Updated.

File: docs/pages/access-controls/idps/saml-grafana.mdx (hunk 2 above)
Comment (alexfornuto) [reply docs/pages/access-controls/idps/saml-grafana.mdx]: Moved.

**Comment by alexfornuto** [comment]
@zmb3 since this is pointed at Mike's branch and it's been approved I'm gonna merge it, and we can continue discussion on titles (if needed) there.

File: docs/config.json [hunk 3]
```diff
@@ -322,6 +322,11 @@
               "slug": "/access-controls/idps/saml-guide/",
               "forScopes": ["enterprise", "cloud"]
             },
+            {
+              "title": "Authenticate to Grafana with Teleport",
```
[the remark this answers is not in the document]
Comment (zmb3) [reply docs/config.json]: Ah, well now that I see it under the "Teleport as an IdP" I think it was probably fine as-is. Apologies, I don't often look at the rendered version when reviewing.

**Pull request merged**